src/test/java/model/OrderTest.java

```diff
@@ -0,0 +1,38 @@
+package model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class OrderTest {
+    @Test
+    void getProduct_ShouldReturn_OrderedProduct_ForTure(){
+        // GIVEN
+        Product product1 = new Product("1", "iphone 14");
+        Order order = new Order("1", product1);
+
+        // WHEN
+        Product actual = order.getProduct();
+
+        // THEN
+        String expectedToStringOfProduct = product1.toString();
+        assertEquals(expectedToStringOfProduct, actual.toString());
+    }
+
+    @Test
+    void getProduct_ShouldReturn_OrderedProduct_ForFalse(){
+        // GIVEN
+        Product product1 = new Product("1", "iphone 14");
+        Order order = new Order("1", product1);
+
+        // WHEN
+        Product actual = order.getProduct();
+
+        // THEN
+        String expectedToStringOfProduct = "Product{" +
```

Comment:
Gibt es einen Grund warum du den String selber baust?
Ist da extra ein Fehler gebaut?
Wenn ja, in der Variable schreiben, wenn nicht, warum nicht einfach product1.toString()?